pom.xml

```diff
@@ -6,7 +6,7 @@
   <modelVersion>4.0.0</modelVersion>
 
    <groupId>ome</groupId>
-   <version>5.2.0</version>
```

Comment:
I've altered the version.  However, the jump is more due to us ignoring and forgetting to do the bumps for the past three bio-formats releases; it should have been 5.2.4 had we kept up to date.

No artefacts; you need to build for your platform.  So unless we want to individually upload platform-specific archives, there's nothing to do.  We can do that sometime in the future if we really care, but it would require work to make it package properly and upload all the separate bits to artifactory.